.gitignore

```diff
@@ -3,3 +3,6 @@ venv
 __pycache__
 .idea
```

Comment:
needs to be `.idea*`

Reply:
.idea or idea/ ignores it, * isn't needed. All the .idea files in this branch are excluded.